ern-runner-gen-ios/test/regen-fixtures.ts

```diff
@@ -16,6 +16,7 @@ getRunnerGeneratorForPlatform('ios')
     },
     mainMiniAppName: 'dummy',
     outDir: pathToGeneratedFixtures,
+    reactNativeVersion: '0.60.5',
```

Comment:
Just wondering: Was this one necessary to fix the fixtures? What does it do/why do we need it?

Reply:
To be able to execute the `regen-fixtures` script in `ern-runner-gen-ios` package, we have to supply the mandatory property `reactNativeVersion` of `RunnerGeneratorConfig`.